aiida_pseudo/cli/install.py

```diff
@@ -140,3 +140,109 @@ def cmd_install_sssp(version, functional, protocol, traceback):
         family.set_cutoffs(cutoffs)
 
         echo.echo_success(f'installed `{label}` containing {family.count()} pseudo potentials')
+
+
+@cmd_install.command('pseudodojo')
+@options.VERSION(type=click.Choice(['03', '04']), default='04')
+@options.FUNCTIONAL(type=click.Choice(['pbe', 'pbesol', 'lda']), default='pbe')
+@options.REL(type=click.Choice(['sr', 'sr3plus', 'fr']), default='sr')
+@options.PROTOCOL(type=click.Choice(['standard', 'stringent']), default='standard')
+@options.HINT(type=click.Choice(['low', 'normal', 'high', 'none']), default='none')
+@options.PSEUDO_FORMAT(type=click.Choice(['psp8', 'upf', 'psml']), default='psp8')
+@options.TRACEBACK()
+@decorators.with_dbenv()
+def cmd_install_pseudodojo(version, functional, rel, protocol, hint, pseudo_format, traceback):
+    """Install an PseudoDojo configuration.
+
+    The PseudoDojo configuration will be automatically downloaded from pseudo-dojo.org to create a new
+    `PseudoDojoFamily` subclass based on the correct pseudopotential format.
+    """
+    # pylint: disable=too-many-locals
+    import requests
+    from aiida.common.files import md5_file
+    from aiida.orm import Group, QueryBuilder
+    from aiida_pseudo import __version__
+    from aiida_pseudo.groups.family import (PseudoDojoConfiguration,
+                                            PseudoDojoFamily,
+                                            PseudoDojoPsmlFamily,
+                                            PseudoDojoPsp8Family,
+                                            PseudoDojoUpfFamily)
+
+    from .utils import attempt, create_family_from_archive
+
+    pseudo_format_families = {'psp8': PseudoDojoPsp8Family, 'psml': PseudoDojoPsmlFamily, 'upf': PseudoDojoUpfFamily}
+
+    metadata_urls = {
+        'nc-sr-04_pbe_standard': 'ONCVPSP-PBE-PDv0.4/standard.djson',
+        'nc-sr-04_pbe_stringent': 'ONCVPSP-PBE-PDv0.4/stringent.djson',
+        'nc-sr-04_pbesol_standard': 'ONCVPSP-PBEsol-PDv0.4/standard.djson',
+        'nc-sr-04_pbesol_stringent': 'ONCVPSP-PBEsol-PDv0.4/stringent.djson',
+        # 'nc-fr-04_pbe_standard': 'ONCVPSP-PBE-FR-PDv0.4/standard.djson',  # md5 does not match
```

Comment:
I think in the case of FR, the hint table has to be taken from the GitHub: https://raw.githubusercontent.com/PseudoDojo/ONCVPSP-PBE-FR-PDv0.4/master/standard.djson

Reply:
The md5 hash of the pseudopotential file and the md5 hash listed in the file that you linked don't match. I need to investigate further to find out why and double check what the problem is.

Reply:
Let me know if you can fix this. Otherwise I will contact the Matteo to see if he can fix the problem.

Reply:
@zooks97 the problem should be in [this file on their repo](https://github.com/abinit/pseudo_dojo/blob/master/pseudo_dojo/pseudos/ONCVPSP-PBE-FR-PDv0.4/standard.djson) then correct? If we are sure we are generating the correct md5, can we simply open a PR that corrects the faulty md5?

Reply:
The md5s in their repo are not faulty, they actually refer to slightly different pseudos than what are on the pseudo-dojo site (and the cutoffs in the file also refer to those tweaked pseudos). Not sure the best way to move forward.